dev-docs/source/auth.rst

```diff
@@ -23,13 +23,14 @@ At every level, the ``auth`` field contains a string that is called a *scope*.
 A request's JWT must contain this scope in its ``scopes`` claim
 (see the ``authorization_django`` source code for details)
 to meet an ``auth`` field's requirements.
-The absence of an ``auth`` field marks a publicly available resource.
+The absence of a scope marks a publicly available resource.
```

Comment:
This was correct, isn't it?

Reply:
I thought it wasn't clear enough, but my edit actually made it less clear. Let me try again.

Reply:
Restored to the old text.